### ISSUE TITLE
Dreamweaver Failing to Import

### ISSUE THREAD

**Issue by walkintom** [autopkg/dataJAR-recipes#181]
```
./Adobe2021Importer.py munki
Looking for /Users/removed/Downloads/Adobe*2021* folders ...
1 Adobe 2021 folder found, creating recipe list...
Looking for pkgs...
Found /Users/removed/Downloads/AdobeDreamweaver2021/Build/AdobeDreamweaver2021_Install.pkg...
Found /Users/removed/Downloads/AdobeDreamweaver2021/Build/AdobeDreamweaver2021_Install.pkg...
Running recipe_list: `/Users/removed/Downloads/adobe2021_list.txt`

Running `['/usr/local/bin/autopkg', 'run', '-v', '--recipe-list', '/Users/removed/Downloads/adobe2021_list.txt', '--report-plist', '/Users/removed/Downloads/adobe2021_report.plist']`...
Processing local.munki.AdobeDreamweaver2021...
Adobe2021Versioner
Adobe2021Versioner: install_pkg /Users/removed/Downloads/AdobeDreamweaver2021/Build/AdobeDreamweaver2021_Install.pkg
Adobe2021Versioner: uninstall_pkg /Users/removed/Downloads/AdobeDreamweaver2021/Build/AdobeDreamweaver2021_Uninstall.pkg
Adobe2021Versioner: Processing /Users/removed/Downloads/AdobeDreamweaver2021/Build/AdobeDreamweaver2021_Install.pkg/Contents/Resources/optionXML.xml
Adobe2021Versioner: sap_code: DRWV
Adobe2021Versioner: target_folder: DRWV21.2
Adobe2021Versioner: architecture_type: x64
Adobe2021Versioner: Processing HD installer
Adobe2021Versioner: app_launch: [INSTALLDIR]/Adobe Dreamweaver 2021.app/Contents/MacOS/Dreamweaver
Adobe2021Versioner: app_bundle: Adobe Dreamweaver 2021.app
Adobe2021Versioner: app_path: Adobe Dreamweaver 2021
Adobe2021Versioner: installed_path: /Applications/Adobe Dreamweaver 2021/Adobe Dreamweaver 2021.app
Adobe2021Versioner: display_name: Adobe Dreamweaver 2021
Adobe2021Versioner: zip_file: AdobeDreamweaver21_x64-mul
Adobe2021Versioner: pimx_dir: 1
Adobe2021Versioner: zip_path: /Users/removed/Downloads/AdobeDreamweaver2021/Build/AdobeDreamweaver2021_Install.pkg/Contents/Resources/HD/DRWV21.2/AdobeDreamweaver21_x64-mul.zip
Adobe2021Versioner: Parsing AdobeDreamweaver21_x64-mul failed with: not well-formed (invalid token): line 1, column 0, checking /Users/removed/Downloads/AdobeDreamweaver2021/Build/AdobeDreamweaver2021_Install.pkg/Contents/Resources/HD/DRWV21.2/Application.json
Checking app_json for version details but sap code DRWV, is not within the known list of apps which we know to check via their Application.json
Failed.
Receipt written to /Volumes/Data/AutoPkg/Cache/local.munki.AdobeDreamweaver2021/receipts/local.munki-receipt-20211029-125301.plist

The following recipes failed:
    local.munki.AdobeDreamweaver2021
        Error in local.munki.AdobeDreamweaver2021: Processor: Adobe2021Versioner: Error: Checking app_json for version details but sap code DRWV, is not within the known list of apps which we know to check via their Application.json

Nothing downloaded, packaged or imported.
```

**Comment by rrwitt** [comment]
Echoing this and getting similar issues with Premiere Pro and After Effects

**Comment by macmule** [comment]
@walkintom this should be sorted now for Dreamweaver.. please repo update

@rrwitt I cannot recreate with After Effects 2021.. will test with Premiere in a sec

**Comment by macmule** [comment]
@rrwitt Premiere Pro worked for me:

```
The following new items were imported into Munki:
    Name                          Version  Catalogs    Pkginfo Path                                                                             Pkg Repo Path                                                                                      Icon Repo Path  
    ----                          -------  --------    ------------                                                                             -------------                                                                                      --------------  
    AdobePremierePro2021-silicon  15.4.1   production  Adobe 2021/Adobe/AdobePremierePro2021-silicon/AdobePremierePro2021-silicon-15.4.1.plist  Adobe 2021/Adobe/AdobePremierePro2021-silicon/AdobePremierePro2021-silicon_Install-ARM-15.4.1.dmg         
```

**Comment by walkintom** [comment]
Dreamweaver working as expected :) cheers.

```
./Adobe2021Importer.py munki
Looking for /Users/t1casetom/Downloads/Adobe*2021* folders ...
1 Adobe 2021 folder found, creating recipe list...
Looking for pkgs...
Found /Users/t1casetom/Downloads/AdobeDreamweaver2021/Build/AdobeDreamweaver2021_Install.pkg...
Found /Users/t1casetom/Downloads/AdobeDreamweaver2021/Build/AdobeDreamweaver2021_Install.pkg...
Running recipe_list: `/Users/t1casetom/Downloads/adobe2021_list.txt`

Running `['/usr/local/bin/autopkg', 'run', '-v', '--recipe-list', '/Users/t1casetom/Downloads/adobe2021_list.txt', '--report-plist', '/Users/t1casetom/Downloads/adobe2021_report.plist']`...
Processing local.munki.AdobeDreamweaver2021...
Adobe2021Versioner
Adobe2021Versioner: install_pkg /Users/t1casetom/Downloads/AdobeDreamweaver2021/Build/AdobeDreamweaver2021_Install.pkg
Adobe2021Versioner: uninstall_pkg /Users/t1casetom/Downloads/AdobeDreamweaver2021/Build/AdobeDreamweaver2021_Uninstall.pkg
Adobe2021Versioner: Processing /Users/t1casetom/Downloads/AdobeDreamweaver2021/Build/AdobeDreamweaver2021_Install.pkg/Contents/Resources/optionXML.xml
Adobe2021Versioner: sap_code: DRWV
Adobe2021Versioner: target_folder: DRWV21.2
Adobe2021Versioner: architecture_type: x64
Adobe2021Versioner: Processing HD installer
Adobe2021Versioner: app_launch: [INSTALLDIR]/Adobe Dreamweaver 2021.app/Contents/MacOS/Dreamweaver
Adobe2021Versioner: app_bundle: Adobe Dreamweaver 2021.app
Adobe2021Versioner: app_path: Adobe Dreamweaver 2021
Adobe2021Versioner: installed_path: /Applications/Adobe Dreamweaver 2021/Adobe Dreamweaver 2021.app
Adobe2021Versioner: display_name: Adobe Dreamweaver 2021
Adobe2021Versioner: zip_file: AdobeDreamweaver21_x64-mul
Adobe2021Versioner: pimx_dir: 1
Adobe2021Versioner: zip_path: /Users/t1casetom/Downloads/AdobeDreamweaver2021/Build/AdobeDreamweaver2021_Install.pkg/Contents/Resources/HD/DRWV21.2/AdobeDreamweaver21_x64-mul.zip
Adobe2021Versioner: Parsing AdobeDreamweaver21_x64-mul failed with: not well-formed (invalid token): line 1, column 0, checking /Users/t1casetom/Downloads/AdobeDreamweaver2021/Build/AdobeDreamweaver2021_Install.pkg/Contents/Resources/HD/DRWV21.2/Application.json
Adobe2021Versioner: app_version: 21.2.0.15523
Adobe2021Versioner: app_bundle: /Applications/Adobe Dreamweaver 2021/Adobe Dreamweaver 2021.app
Adobe2021Versioner: additional_pkginfo: {'minimum_os_version': '10.14.0', 'display_name': 'Adobe Dreamweaver 2021', 'installs': [{'CFBundleShortVersionString': '21.2.0.15523', 'path': '/Applications/Adobe Dreamweaver 2021/Adobe Dreamweaver 2021.app', 'type': 'application', 'version_comparison_key': 'CFBundleShortVersionString', 'CFBundleIdentifier': 'com.adobe.dreamweaver-18.1'}], 'supported_architectures': ['x86_64']}
DmgCreator
DmgCreator: Created dmg from /Users/t1casetom/Downloads/AdobeDreamweaver2021/Build/AdobeDreamweaver2021_Install.pkg at /Volumes/Data/AutoPkg/Cache/local.munki.AdobeDreamweaver2021/AdobeDreamweaver2021_Install-Intel-21.2.0.15523.dmg
DmgCreator
DmgCreator: Created dmg from /Users/t1casetom/Downloads/AdobeDreamweaver2021/Build/AdobeDreamweaver2021_Uninstall.pkg at /Volumes/Data/AutoPkg/Cache/local.munki.AdobeDreamweaver2021/AdobeDreamweaver2021_Uninstall-Intel-21.2.0.15523.dmg
MunkiPkginfoMerger
MunkiPkginfoMerger: Merged {'minimum_os_version': '10.14.0', 'display_name': 'Adobe Dreamweaver 2021', 'installs': [{'CFBundleShortVersionString': '21.2.0.15523', 'path': '/Applications/Adobe Dreamweaver 2021/Adobe Dreamweaver 2021.app', 'type': 'application', 'version_comparison_key': 'CFBundleShortVersionString', 'CFBundleIdentifier': 'com.adobe.dreamweaver-18.1'}], 'supported_architectures': ['x86_64']} into pkginfo
MunkiPkginfoMerger
MunkiPkginfoMerger: Merged {'version': '21.2.0.15523', 'installer_type': ''} into pkginfo
MunkiImporter
MunkiImporter: Using repo lib: AutoPkgLib
MunkiImporter:         plugin: FileRepo
MunkiImporter:           repo: /Volumes/Data/munki_repo
MunkiImporter: Copied pkginfo to: /Volumes/Data/munki_repo/pkgsinfo/apps/Adobe/AdobeDreamweaver2021-21.2.0.15523.plist
MunkiImporter:            pkg to: /Volumes/Data/munki_repo/pkgs/apps/Adobe/AdobeDreamweaver2021_Install-Intel-21.2.0.15523.dmg
Receipt written to /Volumes/Data/AutoPkg/Cache/local.munki.AdobeDreamweaver2021/receipts/local.munki-receipt-20211105-180829.plist

The following new items were imported into Munki:
    Name                  Version       Catalogs  Pkginfo Path                                        Pkg Repo Path                                                   Icon Repo Path  
    ----                  -------       --------  ------------                                        -------------                                                   --------------  
    AdobeDreamweaver2021  21.2.0.15523  testing   apps/Adobe/AdobeDreamweaver2021-21.2.0.15523.plist  apps/Adobe/AdobeDreamweaver2021_Install-Intel-21.2.0.15523.dmg                  

Report plist saved to /Users/t1casetom/Downloads/adobe2021_report.plist.
```

**Comment by macmule** [comment]
thanks for confirming @walkintom

@rrwitt, if you're still having issues and they are the 2021 titles you've tried with.. please raise another issue
